### DISCUSSION TITLE
How to add items to navbar

### DISCUSSION THREAD

**Discussion by nagyleventeboldizsar** [nuxt/content#2123] [Q&A] [answered]
Hello all, 
Is it possible to tell content that I'd like to show one of my page's title in the NavBar component? 
I tried to read everything and it seems the problem is me, because I wasn't able to find anything useful (at my knowledge level right now).
TLDR; How can i tell one of the page to be shown in my Navbar as a route?

**Comment by nobkd** [comment]
You can add a key to your front-matter and query for it inside your nav bar component

**Reply by nagyleventeboldizsar** [reply]
My bad, I was using docus. I wrote down the solution below.

**Comment by nagyleventeboldizsar** [comment, marked as answer]
So I was using docus, and it looks like you have to set aside:{ level:1 }. Then you can filter them.
